storage/bucket.go

```diff
@@ -416,6 +416,10 @@ type BucketAttrs struct {
 	// This field is read-only.
 	Created time.Time
 
+	// Time at which the bucket was last modified.
```

Comment:
Nit: godocs should start with the name of the field; ie `Updated is the time at which the bucket was last modified`. The linter will complain about this.

Reply:
Done